.travis.yml

```diff
@@ -18,6 +18,8 @@ env:
     - MAKEFLAGS="-j2"
     - PGHOST=localhost
     - RGL_USE_NULL=TRUE # Keeps RGL from complaining it can't find X11
+    - _R_CHECK_LENGTH_1_CONDITION_=verbose
+    -_R_CHECK_LENGTH_1_LOGIC2_=verbose
```

Comment:
I'm fuzzy on how verbose mode works -- Do we also need to do something to explicitly turn it on during the build?

Reply:
‘verbose’ prints very detailed diagnostic message
setting it to TRUE will give an error, default is warning.

So I think we are good, we might want to set it explicitly to true, to make sure, but I would like to get a vote on that.

See https://cran.r-project.org/doc/manuals/r-devel/R-ints.html

Reply:
My vote would be to set this to `true`. This can introduce some really insidious bugs, and I've seen it in a lot of places in PEcAn code (especially relative newcomers to R).

Reply:
setting them to true, to see if anything fails.

Reply:
Aah OK, I was interpreting it as something like "behavior depends on the setting of options( "verbose"), but I see I was wrong. Yeah, I'll vote `true` as well.